internal/linters/shell/shellcheck/shellcheck.go

```diff
@@ -0,0 +1,167 @@
+package shellcheck
+
+import (
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/qiniu/reviewbot/internal/linters"
+	"github.com/qiniu/x/log"
+	"github.com/qiniu/x/xlog"
+)
+
+var lintName = "shellcheck"
+
+func init() {
+	linters.RegisterPullRequestHandler(lintName, shellcheckHandler)
+}
+
+func shellcheckHandler(log *xlog.Logger, a linters.Agent) error {
+	/*
+	 * Shellcheck can not support check directories recursively, it is usually used in conjunction with the "find" command
```

Comment:
if so， we can just check the shell files in PR, Refer: https://github.com/qiniu/reviewbot/blob/master/internal/linters/linters.go#L98

Reply:
updated in line 67 to 75